components/automate-pg-gateway/habitat/config/haproxy.conf

```diff
@@ -55,10 +55,10 @@ backend automate_postgresql
             {{~#if ../cfg.resolvers.nameservers }}
     server-template {{node.address}} 8 {{node.address}}:{{node.port}} check resolvers pgdns init-addr none  resolve-prefer ipv4
             {{else}}
-    server {{node.address}} {{node.address}}:{{node.port}} maxconn {{../cfg.max_connections}} check port 6432
+    server {{node.address}} {{node.address}}:{{node.port}} maxconn {{../cfg.max_connections}}  {{~#if ../cfg.chef_manage_db }} check port 6432 {{~/if}} 
```

Comment:
can we put the maxconn into the if condition, since earlier the maxconn was not present for customer managed database